Tests/InternalTests/WorkerQueue/Workers/LegacyWorkerTests.swift

```diff
@@ -0,0 +1,30 @@
+import XCTest
+import AblyAssetTrackingInternalTesting
+import AblyAssetTrackingInternal
+
+class LegacyWorkerTests: XCTestCase
+{
+    private let logHandler = InternalLogHandlerMock.configured
+    private let properties = WorkerQueuePropertiesMock()
+    private var called = false
```

Comment:
Can't this be a local variable in the test function?

Reply:
8a90707